main/SQLHelper.cpp

```diff
@@ -4017,8 +4035,11 @@ void CSQLHelper::UpdateRainLog()
 			if (splitresults.size()<2)
 				continue; //impossible
 
-			int rate=atoi(splitresults[0].c_str());
-			float total = static_cast<float>(atof(splitresults[1].c_str()));
+			float total=0;
+			int rate=0;
+
+			rate=atoi(splitresults[0].c_str());
+			total = static_cast<float>(atof(splitresults[1].c_str()));
```

Comment:
revert these changes please, they are not needed and only give cppcheck issues
This patch is about energy/power anyway ;)